### PR TITLE
[improvement](spill) fuzzy spill and fix bugs

### PR DESCRIPTION
**Pull request by jacktengg**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run builall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run p0

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run p0

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by mrhhsg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run p0

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run p0

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java [hunk 1]
```diff
@@ -1972,7 +1972,6 @@ public void initFuzzyModeVariables() {
         this.topnOptLimitThreshold = (int) Math.pow(10, random.nextInt(5));
 
         // for spill to disk
-        /*
         if (Config.pull_request_id > 10000) {
             if (Config.pull_request_id % 2 == 1) {
                 this.enablePipelineXEngine = true;
```
Comment (Gabriel39) [inline fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java:1977]: Just use default value.

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]

**Review by Gabriel39 — APPROVED** [review]

**Comment by jacktengg** [comment]
run performance

**Pull request merged**